crates/flux-middle/src/rty/pretty.rs

```diff
@@ -440,6 +440,9 @@ impl Pretty for BaseTy {
             BaseTy::Infer(ty_vid) => {
                 w!(cx, f, "{ty_vid:?}")
             }
+            BaseTy::Foreign(def_id) => {
+                w!(cx, f, "{def_id:?}")
```

Comment:
When using `w!` variables cannot be inlined, otherwise we won't use the `Pretty` implementation

```suggestion
                w!(cx, f, "{:?}", def_id)
```

Reply:
What is special about ty_vid such that it can be used inline?